### DISCUSSION TITLE
2023/04/20/obsidian-git-ios-%E5%A4%9A%E5%B9%B3%E5%8F%B0%E5%90%8C%E6%AD%A5/

### DISCUSSION THREAD

**Discussion by giscus** [CasonMo/comment-utterances#10] [Announcements]
# 2023/04/20/obsidian-git-ios-%E5%A4%9A%E5%B9%B3%E5%8F%B0%E5%90%8C%E6%AD%A5/

Obsidian 提供了多种数据同步方式，其中可以通过第三方插件实现 Git 同步。通过 Git 同步，你可以将 Obsidian 中的笔记数据上传到 GitHub、GitLab、Bitbucket 等 Git 仓库中，从而实现在 iOS 和 Mac 等不同设备之间同步笔记。 以下是实现 Obsidian Git 同步的步骤： Mac： 在你的 Git 仓库中创建一个新的仓库，并将其克隆到本地计算

https://cason.work/2023/04/20/obsidian-git-ios-%E5%A4%9A%E5%B9%B3%E5%8F%B0%E5%90%8C%E6%AD%A5/

**Comment by cmgzn** [comment]
太强了，完美解决
